### ISSUE TITLE
Improve simple --help output

### ISSUE THREAD

**Issue by pde** [certbot/certbot#3861]
Brainstorming about verbs, we decided to improve the top level output of `cerbot -h`:

```
certbot --help

  certbot [SUBCOMMAND] [options] [-d domain] [-d domain] ...

Certbot can obtain and install HTTPS/TLS/SSL certificates.  By default,
it will attempt to use a webserver both for obtaining and installing the
cert. The most common SUBCOMMANDS and flags are:
    
obtain, install, and renew certificates:
  (default) run       Obtain & install a cert in your current webserver
  certonly             Obtain or renew a cert, but do not install it
  renew                Renew all previously obtained certs that are near expiry

--apache          Use the Apache plugin for authentication & installation
--standalone      Run a standalone webserver for authentication
--nginx           Use the Nginx plugin for authentication & installation
--webroot         Place files in a server's webroot folder for authentication
--manual          Obtain certs interactively, or using shell script hoooks
 -n               Run non-interactively

manage certificates:    
  certificates         Display information about certs configured with Certbot
  manage             Interactively manage your certificates
  rename              Update a certificate's name 
  revoke               Revoke a previously obtained certificate
    
manage your account with Let's Encrypt:
  register           Create an account
  account           Interactively manage your account

More detailed help:

  -h, --help [topic]    print this message, or detailed help on a topic;
                              the available topics are:

   all, automation, commands, paths, security, testing, or any of the subcommands or
   plugins (certonly, renew, install, register, nginx, apache, standalone,
   webroot, script, etc.)
```

**Comment by pde** [comment]
Closed by: #3863

**Comment by pde** [comment]
Closed by: #3883.
